admin/client/App/screens/List/components/ItemsTable/ItemsTableRow.js

```diff
@@ -38,7 +38,7 @@ const ItemsRow = React.createClass({
 		});
 		// item fields
 		var cells = this.props.columns.map((col, i) => {
-			var ColumnType = Columns[col.type] || Columns.__unrecognised__;
+			var ColumnType = Columns[col.type] || Columns.text;
```

Comment:
Yeah, there used to be an `__unrecognised__` type, which would warn that an implementation was missing. Looks like it was lost - I think we should add it back again, rather than quietly defaulting to `text`. Having a type without an implementation is an error case, and means something's gone wrong at a really low level.